### PR TITLE
Update to iD v1.7.0

### PR DESCRIPTION
**Pull request by bhousel**
trying again..
@tomhughes @jfirebaugh

### CODE REVIEW

File: app/views/site/id.html.erb [hunk 1]
```diff
@@ -19,8 +19,11 @@
     <% locale = select_locale(ID::LOCALES).to_s %>
 
     var id = iD()
+      .presets(iD.data.presets)
+      .imagery(iD.data.imagery)
+      .taginfo(iD.taginfo())
       .embed(true)
-      .assetPath("/iD/") <%# Can't use asset_path('iD/') in production. %>
+      .assetPath("/iD/") <%# Cant use asset_path('iD/') in production. %>
```
Comment (tomhughes) [inline app/views/site/id.html.erb]: Was there a reason for removing the apostrophe here?

**Comment by tomhughes** [comment]
So that works fine, but iD is still saying `1.6.3` in the bottom corner...

File: app/views/site/id.html.erb (hunk 1 above)
Comment (bhousel) [inline app/views/site/id.html.erb]: Just because it was wrecking the syntax highlighting in the rest of the file.

Comment (tomhughes) [inline app/views/site/id.html.erb]: Ha, I obviously have a better editor than you (insert standard editor war here)

**Comment by bhousel** [comment]
> but iD is still saying 1.6.3 in the bottom corner...

oops! missed that one, but it should be right now...

File: app/views/site/id.html.erb (hunk 1 above)
Comment (bhousel) [inline app/views/site/id.html.erb]: hehe, you have a better editor than Github too!

**Comment by bhousel** [comment]
thanks everyone!

**Pull request merged**